src/addserver.html

```diff
@@ -3,7 +3,7 @@
         <form class="addServerForm" style="margin: 0 auto;">
             <h1>${HeaderConnectToServer}</h1>
             <div class="inputContainer">
-                <input is="emby-input" type="url" id="txtServerHost" required="required" label="${LabelServerHost}"/>
+                <input is="emby-input" type="text" id="txtServerHost" required="required" label="${LabelServerHost}"/>
```

Comment:
In this case, we should also revert `autocomplete="off" spellcheck="false" autocapitalize="none" autocorrect="off"`
cc @MrTimscampi